ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -262,12 +266,15 @@ private void terminateCluster(final Command command) {
   CommandRunnerStatus checkCommandRunnerStatus() {
     final Pair<QueuedCommand, Instant> currentCommand = currentCommandRef.get();
     if (currentCommand == null) {
-      return CommandRunnerStatus.RUNNING;
+      return lastPollTime.get() == null ||
+          Duration.between(lastPollTime.get(), clock.instant()).toMillis()
+              < NEW_CMDS_TIMEOUT.toMillis() * 3
+              ? CommandRunnerStatus.RUNNING : CommandRunnerStatus.ERROR;
```

Comment:
Can we just get rid of this and set the status to ERROR if we haven't polled in a while?

Reply:
We only start polling after we `processPriorCommands` so this could still be useful in the case of the CommandRunner getting stuck on start up.